<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPacketList</class>
 <widget class="QMainWindow" name="QtPacketList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet List</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>717</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="packetListTab">
     <attribute name="title">
      <string>Packet List</string>
     </attribute>
     <widget class="QGroupBox" name="packetInGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Packet In</string>
      </property>
      <widget class="QTreeWidget" name="packetInList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>571</width>
         <height>291</height>
        </rect>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="packetOutGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Packet Out</string>
      </property>
      <widget class="QTreeWidget" name="packetOutList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>571</width>
         <height>291</height>
        </rect>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>665</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLine">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>665</y>
        <width>311</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="searchLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>665</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="searchCaseBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>665</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Case Sensitive</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sendCommandTab">
     <attribute name="title">
      <string>Send Packet</string>
     </attribute>
     <widget class="QLineEdit" name="sendCommandLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sendSchidSpin">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="sendSchidLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ServerConnectionHandlerID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="parseFieldEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>591</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="parseFieldsButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Parse Fields</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="parseFieldsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>591</width>
        <height>291</height>
       </rect>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPacketList</receiver>
   <slot>updateLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>packetInList</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>QtPacketList</receiver>
   <slot>copyTextToClipboard(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>packetOutList</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>QtPacketList</receiver>
   <slot>copyTextToClipboard(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPacketList</receiver>
   <slot>sendCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendCommandLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtPacketList</receiver>
   <slot>checkSendButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtPacketList</receiver>
   <slot>updateLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>708</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchCaseBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>QtPacketList</receiver>
   <slot>updateLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>708</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseFieldEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtPacketList</receiver>
   <slot>checkParseFieldsButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parseFieldsButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPacketList</receiver>
   <slot>parseFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateLists()</slot>
  <slot>copyTextToClipboard(QTreeWidgetItem*)</slot>
  <slot>sendCommand()</slot>
  <slot>checkSendButton()</slot>
  <slot>checkParseFieldsButton()</slot>
  <slot>parseFields()</slot>
 </slots>
</ui>
